packages/dma-library/src/strategies/aave-like/multiply/common/build-flashloan-sim-args.ts

```diff
@@ -1,23 +1,29 @@
 import { FEE_BASE } from '@dma-common/constants'
 import * as StrategyParams from '@dma-library/types/strategy-params'
+import { WithFlashLoanArgs } from '@dma-library/types/strategy-params'
+import { IPositionTransitionParams } from '@domain'
 import BigNumber from 'bignumber.js'
 
 /**
  * TODO: Remove the need for this function
  * Flashloan calculations should not be part of the adjustToTargetRiskRatio logic
  */
-export function buildFlashloanSimArgs(
-  flashloanTokenAddress: string,
+export async function buildFlashloanSimArgs(
```

Comment:
Why is this `async`? I don't see any calls that need `await`

Reply:
My apologies, I forgot to remove it after conducting my tests.